azure-pipelines.yml

```
@@ -32,8 +32,7 @@ jobs:
                   --ignore=tests/licensedcode/test_detection_datadriven1.py \
                   --ignore=tests/licensedcode/test_detection_datadriven2.py \
                   --ignore=tests/licensedcode/test_detection_datadriven3.py \
-                  --ignore=tests/licensedcode/test_detection_datadriven4.py \
-                  tests/licensedcode
+                  --ignore=tests/licensedcode/test_detection_datadriven4.py
```

Why removing this? this will be a regression.

```suggestion
                  --ignore=tests/licensedcode/test_detection_datadriven4.py \
                  --ignore=tests/licensedcode/test_additional_license.py \
                   tests/licensedcode
```

---

Yes, adding this back.

---

Added this back.